deps/rabbitmq_shovel_management/src/rabbit_shovel_mgmt.erl

```diff
@@ -117,6 +114,34 @@ filter_vhost_user(List, _ReqData, #context{user = User = #user{tags = Tags}}) ->
                          VHost     -> lists:member(VHost, VHosts)
                      end].
 
+restart_shovel(VHost, Name, Node) ->
+      case rpc:call(Node, rabbit_shovel_util, restart_shovel, [VHost, Name], infinity) of
+        {badrpc, Reason} ->
+          rabbit_log:error("rabbit_shovel_util Could not restart_shovel ~s in vhost ~s Reason: ~s", [Name, VHost, Reason]),
```

Comment:
You're changing the log message format. I think this looks better and is more meaningful. But, consider log format changes and code that relies on presence of certain logs. During the shovel security incident, I think we were looking for this particular shovel error message to understand the extent of the impact.